### PR TITLE
Update content

### PR DESCRIPTION
**Pull request by mbfisher**
@dan-j please have a quick check I haven't done anything stupid!

M

### CODE REVIEW

File: app/static/components/Scene.js [hunk 1]
```diff
@@ -31,6 +31,8 @@ export default class Scene extends React.Component {
     }
 
     onChange(data, isValid) {
+        console.log('Scene received data', data);
```
Comment (dan-j) [inline app/static/components/Scene.js]: Still want console.logs?

Comment (mbfisher) [inline app/static/components/Scene.js]: Not really but this one's really useful so may as well leave it in!

File: app/static/components/StepDetails.js [hunk 2]
```diff
@@ -1,80 +1,171 @@
...
-            stepName: '',
-            stepDescription: '',
+import Textarea from './Textarea'
+import AbstractDetails from './AbstractDetails';
+
+export class MidnightStepDetails extends AbstractDetails {
+    componentWillMount() {
+        this.setDataPrefix('step');
+
+        const value = {
```
Comment (dan-j) [inline app/static/components/StepDetails.js]: declared within the method? if we were being proper i'd say declare once..

File: app/static/scenes/CreateGoal.js [hunk 3]
```diff
@@ -11,9 +11,9 @@ export default function CreateGoal({ csrfToken }) {
     return <Scene
         csrfToken={csrfToken}
         pages={[
-            CustomGoal,
+            CustomGoalDetails,
             GoalTarget,
-            StepDetails,
+            props => <CustomStepDetails {...props} stepNumber={1} />,
```
Comment (dan-j) [inline app/static/scenes/CreateGoal.js]: what's this do? `props => <Cust....`

Comment (mbfisher) [inline app/static/scenes/CreateGoal.js]: Create a stateless component (double arrow function) that renders the page. Let's pass the `stepNumber` prop in, and proxies the `data` and `onChange` props passed to it from the `Scene`.

M

File: app/templates/steps/start.html [hunk 4]
```diff
@@ -43,7 +43,7 @@
 
         <p>Remember: Don’t lose a life so close to the finishing line – track your writing.</p>
 
-        <p>Writing secret #4: Check your inbox now to find out why normalizing your writing habit is key to writing productivity.</p>
+        <p>Writing secret #4: Check your inbox now to find out why normalising your writing habit is essential to writing productivity.</p>
```
Comment (dan-j) [inline app/templates/steps/start.html]: productivity or productively?

File: app/views/goals.py [hunk 5]
```diff
@@ -95,7 +96,10 @@ def new_custom_goal(user, params, start, timezone):
 @login_required
 @is_active
 def new(request):
-    user_is_subscribed = is_user_subscribed(request.user)
+    if 'subscribed' in request.GET:
+        user_is_subscribed = request.GET['subscribed'] == 'true'
```
Comment (dan-j) [inline app/views/goals.py]: we're trusting http headers to decide if subscribed?

Comment (mbfisher) [inline app/views/goals.py]: Nope, just a development hack for now.

M

File: app/static/scenes/CreateGoal.js (hunk 3 above)
Comment (dan-j) [inline app/static/scenes/CreateGoal.js]: nice... well, kind of nice... lots of magic, but nice magic

File: app/templates/steps/start.html (hunk 4 above)
Comment (dan-j) [inline app/templates/steps/start.html]: It makes sense both ways but I kind of have to think about "writing productivity" before it makes sense, if that makes sense? 😂

File: app/views/goals.py (hunk 5 above)
Comment (dan-j) [inline app/views/goals.py]: ROFL-copter

**Comment by dan-j** [comment]
Cool, not much to complain about 👍

**Pull request merged**